schema/home/context.js

```diff
@@ -1,5 +1,6 @@
 import { create, assign } from "lodash"
 import gravity from "lib/loaders/gravity"
+import artistLoader from "lib/loaders/per_type"
```

Comment:
This isn’t correct and would result in a singleton that never releases memory, kinda like the `gravity` Singleton we’re moving away from. Instead accept more arguments like all other `resolve` functions you’ve dealt with. The 4th one is the rootValue where you can extract the instance of the loader specific to the request.